BACnetServerExample/BACnetServerExample.cpp

```diff
@@ -566,7 +574,45 @@ int main(int argc, char** argv)
 		std::cerr << "Failed to add NetworkPort" << std::endl;
 		return -1;
 	}
-	std::cout << "OK" << std::endl;
+	fpSetPropertyEnabled(g_database.device.instance, CASBACnetStackExampleConstants::OBJECT_TYPE_NETWORK_PORT, g_database.networkPort.instance, CASBACnetStackExampleConstants::PROPERTY_IDENTIFIER_BBMD_ACCEPT_FD_REGISTRATIONS, true);
+	fpSetPropertyEnabled(g_database.device.instance, CASBACnetStackExampleConstants::OBJECT_TYPE_NETWORK_PORT, g_database.networkPort.instance, CASBACnetStackExampleConstants::PROPERTY_IDENTIFIER_BBMD_BROADCAST_DISTRIBUTION_TABLE, true);
+	fpSetPropertyEnabled(g_database.device.instance, CASBACnetStackExampleConstants::OBJECT_TYPE_NETWORK_PORT, g_database.networkPort.instance, CASBACnetStackExampleConstants::PROPERTY_IDENTIFIER_BBMD_FOREIGN_DEVICE_TABLE, true);
+	
+	// Ask for BBMD Address, Port, and Mask
```

Comment:
This shouldn't be asked on start up as most people won't have a BBMD to connect to. Instead make this part of the UserInput function